custom/icds_reports/static/icds_reports/js/spec/indie-map.directive.spec.js

```diff
@@ -172,12 +172,10 @@ describe('Indie Map Directive', function () {
         controller.handleMapClick(mockGeography);
         $httpBackend.flush();
 
-        expected = {"location_id": "9951736acfe54c68948225cc05fbbd63", "location_name": "test-id"};
+        expected = {"location_id": "9951736acfe54c68948225cc05fbbd63", "location_name": "Chhattisgarh"};
```

Comment:
It took a while, but I convinced myself that this is a bugfix. 

Previously it used the value from the controller's `geography` (https://github.com/dimagi/commcare-hq/pull/26419/files#diff-fbe8c00bf9eb04c909fea8be321d82a0L354), but now it uses the value from the API result (https://github.com/dimagi/commcare-hq/pull/26419/files#diff-bb36edf32ac1b667bad9d081d4aee3c1R34) - which it was doing in other components and should be more correct.

Reply:
cool, thank you for explaining